documentation/pom.xml

```diff
@@ -20,6 +20,8 @@
     <description>Hibernate Search reference documentation</description>
 
     <properties>
+        <test.elasticsearch.run.skip>false</test.elasticsearch.run.skip>
```

Comment:
Shouldn't there be a `<test.database.run.skip>false</test.database.run.skip>` too? Or maybe we don't run the documentation tests against each database? If so, maybe we should?

Reply:
Yes we don't run, but I think that we should. Let's add `<test.database.run.skip>false</test.database.run.skip>` for consistency.